### PR TITLE
CSHARP-3147: Update default JSON output mode

### PR DESCRIPTION
**Pull request by papafe**

### CODE REVIEW

File: tests/MongoDB.Bson.Tests/BsonExtensionMethodsNominalTypeTests.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@
 using FluentAssertions;
 using MongoDB.Bson.Serialization;
 using System;
+using MongoDB.Bson.IO;
```
Comment (BorisDog) [inline tests/MongoDB.Bson.Tests/BsonExtensionMethodsNominalTypeTests.cs:19]: minor: sort usings

File: tests/MongoDB.Bson.Tests/IO/JsonWriterTests.cs [hunk 2]
```diff
@@ -69,11 +69,20 @@ public void JsonWriter_should_support_writing_multiple_documents(
             }
         }
 
+        [Fact]
+        public void JsonWriter_should_have_releaxed_extendend_json_as_default()
```
Comment (BorisDog) [inline tests/MongoDB.Bson.Tests/IO/JsonWriterTests.cs:73]: typo: `relaxed`

**Review by BorisDog — APPROVED** [review]
LGTM + minor changes requested.

**Review by rstam — APPROVED** [review]
LGTM

**Pull request merged**